src/neither.js

```diff
@@ -1,4 +1,5 @@
 import { compose, complement, either } from 'ramda';
+import curry2 from 'ramda/src/internal/_curry2';
```

Comment:
Can we use `curry` or `curryN` instead of internal `curry2`. We only use `curry1` from internals and probably get rid of it in the future and replace it with `curryN` so that we're dependant on internal api or ramda as few as possible.

Reply:
OK, I used curry instead of curry2